rotary/rules/rotary.smk

```diff
@@ -518,16 +538,14 @@ rule short_read_contamination_filter:
         "{sample}/benchmarks/qc/short/short_read_contamination_filter.benchmark.txt"
     params:
         contamination_filter_kmer_length = config.get("contamination_filter_kmer_length"),
-        all_contaminant_references = ','.join(CONTAMINANT_REFERENCE_GENOMES_NCBI +
-            config.get("contamination_references_custom_filepaths"))
     threads:
         config.get("threads", 1)
     resources:
         mem = config.get("memory")
     shell:
         """
         bbduk.sh -Xmx{resources.mem}g threads={threads} in={input.short_r1} in2={input.short_r2} \
-          ref={params.all_contaminant_references} out={output.short_r1} out2={output.short_r2} \
+          ref={input.ncbi_contamination_references} {input.custom_contamination_references} out={output.short_r1} out2={output.short_r2} \
```

Comment:
@jmtsuji Does bbduk need `,` between the file paths if there are multiple input files?

Reply:
Indeed, commas are needed based on reading the docs

Reply:
Fixed in https://github.com/jmtsuji/rotary/pull/82/commits/c6ff6557d46609d5fd28edf005c4bddaa22218f0